webui/react/package.json

```diff
@@ -29,6 +28,7 @@
     "io-ts": "^2.2.4",
     "js-sha512": "^0.8.0",
     "plotly.js-basic-dist": "^1.54.1",
+    "portable-fetch": "^3.0.0",
```

Comment:
question: Just checking to see if this was still needed. Not seeing any references to it.

Reply:
the generated code needs it 
![Screen Capture_select-area_20200706142942](https://user-images.githubusercontent.com/12127420/86647760-2a5d2080-bf95-11ea-85e8-ae7c78930416.png)